build-test.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
+# For scenario testing purposes
 
-./build.sh
```

Comment:
No need to specify base build. Because docker will understand the requirements from the stage declaration